src/components/ContactForm/ContactForm.tsx

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import { useState } from 'react';
 import styles from './ContactForm.module.css';
 import { createShortId } from '../../serviсes/shortId';
+import ContactFormProps from '../../interfaces/ContactForm.interface';
```

Comment:
move `ContactFormProps ` into this file

Reply:
All interfaces including this one have been moved to their own files.